poem.html

```diff
@@ -0,0 +1,208 @@
+			  <ul class="dropdown-menu">
+				<li><a href="#">Subject Matter</a></li>
+				<li><a href="#">Aurther</a></li>
+				<li><a href="#">Something else here</a></li>
+				<li role="separator" class="divider"></li>
+				<li><a href="#">Separated link</a></li>
+				<li role="separator" class="divider"></li>
+				<li><a href="#">One more separated link</a></li>
+			  </ul>
+			</li> -->
+		  </ul>	
+          <ul class="nav navbar-nav navbar-right">
+        		<li><a href="#"><span class="glyphicon glyphicon-user" aria-hidden="true"></span> Sign In</a></li>
+      	  </ul>  
+        </div><!--/.navbar-collapse -->
+      </div>
+    </nav>
+
+    <!-- Main jumbotron for a primary marketing message or call to action -->
+    <div class="jumbotron">
+      <div class="container poem-pg-poem-title">
+        <h1>The Road Not Taken</h1>— by Robert Frost<p></p>
+        <p></p>
+        
+      </div>
+    </div>
+
+    <div class="container container-home">
+      <!-- Example row of columns -->
+      <div class="row">
+        <div class="col-md-4">
+			  <hr>
+          <!--  <img src="./img/360px-Jb_modern_frost_2_e%20(1).jpg" alt="Author Img" height="42" width="42"> -->
+            <img src="./img/360px-Jb_modern_frost_2_e%20(1).jpg" class="img-responsive" alt="Author Image">
+
+
+            <hr>
+              <p class="poem-pg-author-bio">Robert Frost was born in San Francisco, California, to journalist William Prescott Frost, Jr., and Isabelle Moodie. His mother was a Scottish immigrant, and his father descended from Nicholas Frost of Tiverton, Devon, England, who had sailed to New Hampshire in 1634 on the Wolfrana.</p>
+        </div>
+        <div class="col-md-8 "poem-pg-poem-text">
+          <h2 class="visible-md-block visible-lg-block"></h2>
+				
+          <p>
+            Two roads diverged in a yellow wood,<br>
+            And sorry I could not travel both<br>
+            And be one traveler, long I stood<br>
+            And looked down one as far as I could<br>
+            To where it bent in the undergrowth;<br>
+            <br>
+            Then took the other, as just as fair,<br>
+            And having perhaps the better claim,<br>
+            Because it was grassy and wanted wear;<br>
+            Though as for that the passing there<br>
+            Had worn them really about the same,<br>
+            <br>
+            And both that morning equally lay<br>
+            In leaves no step had trodden black.<br>
+            Oh, I kept the first for another day!<br>
+            Yet knowing how way leads on to way,<br>
+            I doubted if I should ever come back.<br>
+            <br>
+            I shall be telling this with a sigh<br>
+            Somewhere ages and ages hence:<br>
+            Two roads diverged in a wood, and I—<br>
+            I took the one less traveled by,<br>
+            And that has made all the difference.<br>
+        </p>
+
```

Comment:
Add a comment here so it's clear that we change from the poem section to the review. I suggest a 'Reader reviews' comment in a new line.